include/kernel/stateMachine.h

```diff
@@ -0,0 +1,487 @@
+/*
```

Comment:
I am assuming this file is a copy of the original, therefore we will not want to change anything in it for now.  If this is true, will not review this file and will leave as is.

Reply:
Yes.